### DISCUSSION TITLE
Difference between daily_vaccinations and daily_people_vaccinated?

### DISCUSSION THREAD

**Discussion by Akashkunwar** [owid/covid-19-data#2589] [Q&A] [answered]
Can anyone tell me what is difference between daily_vaccinations and daily_people_vaccinated?
in the below dataset.

https://raw.githubusercontent.com/owid/covid-19-data/master/public/data/vaccinations/vaccinations.csv

**Comment by lucasrodes** [comment, marked as answer]
Hi @Akashkunwar,
Thanks for your interest in the data.

The main difference is that `daily_vaccinations` counts doses while `daily_people_vaccinated` counts people. You can find the definitions of the cited file in [this file](https://github.com/owid/covid-19-data/tree/master/public/data/vaccinations). Find below the definitions for these metrics:


* `daily_vaccinations`: new doses administered per day (7-day smoothed). For countries that don't report data on a daily basis, we assume that doses changed equally on a daily basis over any periods in which no data was reported. This produces a complete series of daily figures, which is then averaged over a rolling 7-day window. An example of how we perform this calculation can be found [here](https://github.com/owid/covid-19-data/issues/333#issuecomment-763015298).
* `daily_people_vaccinated`: daily number of people receiving a first COVID-19 vaccine dose (7-day smoothed).

**Reply by Akashkunwar** [reply]
Thanks! it really helps.
